specs/endpoint-security.spec.yml

```diff
@@ -90,3 +88,26 @@ inputs:
       log:
         path: "C:\\Program Files\\Elastic\\Endpoint\\state\\log\\endpoint-*.log"
       operations: *operations
+  - name: endpoint
+    description: "Endpoint Security"
+    platforms:
+      - container/amd64
+      - container/arm64
+    outputs:
+      - elasticsearch
+      - logstash
+      - kafka
+    proxied_actions:
```

Comment:
should these still be proxied in a container?

if so, use a reference as well

Reply:
Roger that!